service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/QuestionnaireResponseMapper.java

```diff
@@ -0,0 +1,80 @@
+import org.hl7.fhir.dstu3.model.QuestionnaireResponse.QuestionnaireResponseItemComponent;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.hl7.fhir.dstu3.model.StringType;
+import org.nhspathways.webservices.pathways.pathwayscase.PathwaysCaseDocument.PathwaysCase;
+import org.nhspathways.webservices.pathways.pathwayscase.PathwaysCaseDocument.PathwaysCase.PathwayDetails.PathwayTriageDetails.PathwayTriage.TriageLineDetails.TriageLine;
+import org.nhspathways.webservices.pathways.pathwayscase.PathwaysCaseDocument.PathwaysCase.PathwayDetails.PathwayTriageDetails.PathwayTriage.TriageLineDetails.TriageLine.Question;
+import org.nhspathways.webservices.pathways.pathwayscase.PathwaysCaseDocument.PathwaysCase.PathwayDetails.PathwayTriageDetails.PathwayTriage.TriageLineDetails.TriageLine.Question.Answers.Answer;
+import org.springframework.stereotype.Component;
+
+import lombok.AllArgsConstructor;
+import uk.nhs.adaptors.oneoneone.cda.report.util.DateUtil;
+
+@Component
+@AllArgsConstructor
+public class QuestionnaireResponseMapper {
+
+    private final QuestionnaireMapper questionnaireMapper;
+
+    public QuestionnaireResponse mapQuestionnaireResponse(PathwaysCase pathwaysCase, Reference patient, Reference encounter,
+        TriageLine triageLine) {
+        QuestionnaireResponse questionnaireResponse = new QuestionnaireResponse();
+        Questionnaire questionnaire = questionnaireMapper.mapQuestionnaire(pathwaysCase, triageLine);
+
+        questionnaireResponse.setIdElement(newRandomUuid());
+
+        questionnaireResponse
+            .setQuestionnaire(new Reference(questionnaire))
+            .setQuestionnaireTarget(questionnaire)
+            .setStatus(QuestionnaireResponse.QuestionnaireResponseStatus.COMPLETED)
+            .setSubject(patient)
+            .setContext(encounter);
+
+        if (pathwaysCase.getCaseDetails() != null) {
+            if (pathwaysCase.getCaseDetails().isSetCaseId()) {
+                questionnaireResponse.setIdentifier(new Identifier().setValue(pathwaysCase.getCaseDetails().getCaseId()));
+            }
+        }
+        if (pathwaysCase.isSetCaseReceiveEnd()) {
+            questionnaireResponse.setAuthored(DateUtil.parsePathwaysDate(pathwaysCase.getCaseReceiveEnd().toString()));
+        }
+        if (triageLine.getQuestion() != null) {
+            questionnaireResponse.addItem(getItem(triageLine.getQuestion()));
+        }
+
+        return questionnaireResponse;
+    }
+
+    private QuestionnaireResponseItemComponent getItem(Question question) {
+        QuestionnaireResponseItemComponent item = new QuestionnaireResponseItemComponent();
+        QuestionnaireResponseItemAnswerComponent answer = new QuestionnaireResponseItemAnswerComponent();
+        StringType correctAnswerText = new StringType();
+
+        if (question.getQuestionId() == null || question.getQuestionId().isBlank()) {
+            item.setLinkId("N/A");
```

Comment:
Few references of `N/A` throughout multiple files, maybe make into a constant somewhere?